src/lib/auth.ts

```diff
@@ -42,7 +42,7 @@ export const ROLES: {
 
 		`resin.service_environment_variable.get?service/any(s:s/${applicationControlsDevice})`,
 
-		`resin.device_service_environment_variable.get?service_install/any(si:si/installs__service/any(s:s/${applicationControlsDevice}))`,
+		`resin.device_service_environment_variable.all?service_install/any(si:si/installs__service/any(s:s/${applicationControlsDevice}))`,
```

Comment:
hmmm - I don't see a limit to the specific device here @imrehg

Reply:
I don't think this is the permission to be changed.

Reply:
Indeed, actually, need to create a new permission line.